peps/pep-0740.rst

```diff
@@ -72,12 +71,10 @@ Additionally, this proposal identifies the following motivations:
   the index has no mechanism for identifying the right public key for the
   signature.
 
-While this PEP does not recommend a specific digital attestation format,
-it does recognize the utility of Trusted Publishing as a pre-existing,
-"zero-configuration" source of strong provenance for Python packages.
-Consequently this PEP includes a proposed scheme for exposing each release
-file's Trusted Publisher metadata, with the expectation that a future digital
-attestation format will likely make use of it.
+This PEP recommends a generic attestation format, including a serializable
+payload format for signature generation, with the expectation that adopt the
```

Comment:
This sentence is missing a subject. I don't understand the topic well enough to know what is meant, but maybe:
```suggestion
payload format for signature generation, with the expectation that index providers adopt the
```

Reply:
Thanks! Yeah, that's the intended subject.